prusti-interface/src/trait_register.rs

```diff
@@ -1,246 +0,0 @@
-// © 2020, ETH Zurich
-//
-// This Source Code Form is subject to the terms of the Mozilla Public
-// License, v. 2.0. If a copy of the MPL was not distributed with this
```

Comment:
@cmatheja If I am not mistaken, you are supervising @jakobbeckmann, right? I think it would be better to fix the broken functionality instead of deleting it.

Reply:
Right, I'll give it a look. Thanks for spotting this.